src/clients/python/sams_client/client.py

```diff
@@ -84,20 +86,28 @@ def request(
         base_url = self.config.get('base_url')
         url = f'{base_url}{api}'
         headers = self.auth.apply_headers(headers)
-        response = request(url, headers=headers, data=data, files=files, params=params)
+        response = request(
+            url,
+            headers=headers,
+            data=data,
+            files=files,
+            params=params,
+            external_user_id=external_user_id)
```

Comment:
`external_user_id` should be added to the `params` variable (the requests lib is failing in tests here)